README.md

```diff
@@ -214,17 +224,25 @@ Close the GUI and reopen to confirm the path is correct.
 
 # DeepProfiler Project Setup:
 
-Deep Profiler must be installed via Github.
+DeepProfiler must be installed via Github.
 Commit [`2fb3ed3`](https://github.com/cytomining/DeepProfiler/commit/2fb3ed3027cded6676b7e409687322ef67491ec7) was used while developing `idrstream`.
 Install the repository into `idrstream/` with:
-```sh
+```console
+# make sure that the conda environment `idrstream_dp` is activated
+conda activate idrstream_dp
 cd idrstream/
 git clone https://github.com/cytomining/DeepProfiler.git
 cd DeepProfiler/
-# make sure that the conda environment `idrstream_dp` is activated
 pip install -e .
 ```
 
+Installing this version of DeepProfiler will downgrade numpy to the wrong version, so it is necessary to reinstall numpy with:
+```console
+# make sure that the conda environment `idrstream_dp` is activated
+conda activate idrstream_dp
+pip install numpy==1.23.3
+```
```

Comment:
This seems like an area which could get difficult to navigate, especially as versions change or revisions are made with this repo. Is there a more elegant way to specify these environment details within the .yml environment specifications?